src/v2i-hub/RSUHealthMonitorPlugin/src/RSUConfigurationList.h

```diff
@@ -25,12 +28,17 @@ namespace RSUHealthMonitor
         RSUMIB_V_1218 = 2
     };
 
+    std::string mib_version_to_string(const RSUMibVersion &mibversion);
```

Comment:
I think we can make this function const

Reply:
It is a free function meaning it is not associated with any class or object so it is already const. I don't think I can even add that classifier to the function since it would not make sense. In a member function const before would indicate that the function does not modify the class/object it is a member of but in this case it is not a member of any class/object. That is my understanding.